### ISSUE TITLE
Mess of warnings when creating derivatives

### ISSUE THREAD

**Issue by hackmastera** [chemheritage/chf-sufia#431]
There are about 4 sets of these per derivative; it would be really helpful to silence them.

But I also wonder whether we should make some changes to the data we add to our images? It looks like maybe we're not entirely complying with standards here. @catlu 

```
mogrify: Unknown field with tag 306 (0x132) encountered. `TIFFReadCustomDirectory' @ warning/tiff.c/TIFFWarnings/925.
mogrify: Wrong data type 3 for "PixelXDimension"; tag ignored. `TIFFReadCustomDirectory' @ warning/tiff.c/TIFFWarnings/925.
mogrify: Wrong data type 3 for "PixelYDimension"; tag ignored. `TIFFReadCustomDirectory' @ warning/tiff.c/TIFFWarnings/925.
mogrify: Incompatible type for "FileSource"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/925.
mogrify: Incompatible type for "SceneType"; tag ignored. `TIFFFetchNormalTag' @ warning/tiff.c/TIFFWarnings/925.
```

**Comment by catlu** [comment]
Is this for all our images? I took a look at some of the file info from a tiff, and it seems just like pixel dimension numbers and camera data. I'll check with Jay to see if he can figure any more of what's in there.

**Comment by hackmastera** [comment]
Yes each image gives output like this.

**Comment by catlu** [comment]
I looked at some Exif data that Jay pulled for me, and it seems like our data is correct and according to standard so not sure why there are those errors. Here's a sample from our TIFFs:

         <exif:PixelXDimension>6732</exif:PixelXDimension>
         <exif:PixelYDimension>8984</exif:PixelYDimension>
         <exif:FileSource>3</exif:FileSource>
         <exif:SceneType>1</exif:SceneType>

The first two are the recorded dimensions of the file, and the last two tag numbers are set values for images recorded on a digital still camera. I know these are the tags for the Exif 2.3 standard, so is it possible ImageMagick is using another Exif standard?

**Comment by hackmastera** [comment]
Okay; it seems possible that libtiff just doesn't really know what to do with these fields. we were able to silence the warnings in the logs so if y'all aren't concerned I won't be, either. closed by #432
